Trainee/Reviews/Reviews.jsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { useState } from "react";
+import { FaStar } from "react-icons/fa";
+import axios from "../../../api/axios";
+import ViewReviews from "./ViewReviews/ViewReviews";
+import { Scrollbars } from "react-custom-scrollbars";
+import "./Reviews.css";
+
+const reviewsUrl = "/review";
```

Comment:
Move to URLConstants.js file. Rename variable to REVIEWS_URL. please follow naming conventions.